### ISSUE TITLE
Error in group repositories view

### ISSUE THREAD

**Issue by eladyn** [haiwen/seafile-rpi#50]
Each time I want to see the repositories of a specific group ([https://seafile.example.com/group/1/](#)) there are no repositories displayed and it throws an error. 

[Detailed description](https://forum.seafile.com/t/error-in-seahub-when-accessing-group-shares/9540) and [workaround](https://forum.seafile.com/t/error-in-seahub-when-accessing-group-shares/9540/6)

**Comment by felf** [comment]
I just tried out seafile on a raspi and encountered the same. If you look closely, you will see that it says “Error” at the top. I’m using a freshly-downloaded archive of version 7.0.5.

The log says:
```python
2020-04-17 21:39:32,750 [ERROR] django.request:135 handle_uncaught_exception Internal Server Error: /api/v2.1/groups/1/libraries/
Traceback (most recent call last):
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/Django-1.11.16-py2.7.egg/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/djangorestframework-3.3.3-py2.7.egg/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/thirdpart/djangorestframework-3.3.3-py2.7.egg/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/seahub/api2/endpoints/utils.py", line 42, in _decorated
    return func(view, request, group_id, *args, **kwargs)
  File "/home/pi/seafile/seafile-server-7.0.5/seahub/seahub/api2/endpoints/group_libraries.py", line 106, in get
    if '@seafile_group' in email:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by jobenvil** [comment]
Fair enough. @felf this is already [discussed here](https://forum.seafile.com/t/error-in-seahub-when-accessing-group-shares/9540/6). Thanks mate @fweisser

**Comment by jobenvil** [comment]
Closing, since this is fixed in `v7.1.4`
